misk-actions/src/main/kotlin/misk/exceptions/Exceptions.kt

```diff
@@ -54,6 +54,16 @@ open class WebActionException(
 ) : Exception(message, cause) {
   val isClientError = code in (400..499)
   val isServerError = code in (500..599)
+
+  @Deprecated("Use code instead")
+  val statusCode: StatusCode = run {
```

Comment:
what's the advantage of using `run` over a `get()`?

Reply:
Nothing really, just wanted to have it throw here.